lib/authlogic/session/klass.rb

```diff
@@ -46,22 +46,6 @@ def klass_name
 
       # :nodoc:
       module InstanceMethods
-        # Creating an alias method for the "record" method based on the klass
-        # name, so that we can do:
-        #
-        #   session.user
-        #
-        # instead of:
-        #
-        #   session.record
```

Comment:
Since we're losing these docs, WDYT about a one-liner comment in the constructor? e.g. `# e.g. alias session.user to session.record`.